cloud/disk_manager/test/acceptance/test_runner/base_acceptance_test_runner.py

```diff
@@ -322,12 +322,13 @@ def _find_or_create_eternal_disk(self, disk_name_prefix) -> Ycp.Disk:
         return disk
 
     @contextlib.contextmanager
-    def instance_policy_obtained(self, handler):
+    def instance_policy_obtained(self, handler, keep_instance_on_error=False):
```

Comment:
We do not check keep_instance_on_error when actual error occurs. 
I think we should have the following flow in pseudo code:
if error and keep_instance_on_error:
     self._auto_delete = False

Reply:
If exception is raised - it is, in fact, error

Reply:
https://github.com/ydb-platform/nbs/blob/5f4f11a38f5a9c3f60da5c2aeb2d71a3b71e71a9/cloud/disk_manager/test/acceptance/test_runner/sync_acceptance_test_runner.py#L183